tests/libgit2/refs/revparse.c

```diff
@@ -299,10 +299,14 @@ void test_refs_revparse__ordinal(void)
 	test_object("nope@{0}", NULL);
 	test_object("master@{31415}", NULL);
 	test_object("@{1000}", NULL);
-	test_object("@{2}", NULL);
+	test_object("@{7}", NULL);
 
 	test_object("@{0}", "a65fedf39aefe402d3bb6e24df4d4f5fe4547750");
-	test_object("@{1}", "be3563ae3f795b2b4353bcce3a527ad0a4f7f644");
+	test_object("@{1}", "a4a7dce85cf63874e984719f4fdd239f5145052f");
```

Comment:
Running `git --git-dir=tests/resources/testrepo.git rev-parse @{1}` gives `be3563ae3f795b2b4353bcce3a527ad0a4f7f644` which is consistent with its [documentation](https://git-scm.com/docs/git-rev-parse#Documentation/git-rev-parse.txt-emltngtemegem1em):
> For example, if you are on branch blabla then @{1} means the same as blabla@{1}

Your other changes are consistent with git. I'm testing with git 2.34.1.

Reply:
Fixed.